### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by csmatthews**
I have updated the README for spelling, grammar, and re-organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,48 +73,46 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
-- [eCognition](http://www.ecognition.com/suite/ecognition-developer) - A software for object-based image analysis.
-- [PCI Geomatica](http://www.pcigeomatics.com/software/geomatica/professional) - A remote sensing desktop software package for processing earth observation data. [Commercial]
-- [Google Earth](https://www.google.com/earth/) - is a computer program that renders a 3D representation of Earth based on satellite imagery.
-- [Google Earth Studio](https://www.google.com/earth/studio/) - A powerful earth motion design software.
-- [Opticks](https://opticks.org/) - An expandable remote sensing and imagery analysis software platform.
-- [GRASS GIS](https://grass.osgeo.org/) - Used for geospatial data management and analysis, as a founding member of OSGEO
-- [Orfeo toolbox](https://www.orfeo-toolbox.org/) -  A open-source library for remote sensing image processing.
 
 
-## 3D Application
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 2]
```diff
@@ -73,48 +73,46 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
-- [Google Earth](http://earth.google.com/) - Bringing a earth view for global mapping
+## 3D Applications
+- [ArcGIS Earth](http://www.esri.com/software/arcgis-earth) - Allows you to explore any part of the world. Work with a variety of 3D and 2D map data formatt, including KML. Display data, sketch placemarks, measure and perform interactive analysis, and add annotations.
+- [CityEngine](http://www.esri.com/software/cityengine/) - Advanced 3D modeling software.
+- [Google Earth](http://earth.google.com/) - A computer program that renders a 3D representation of Earth based on satellite imagery.
+- [Skyline](http://www.skylineglobe.com/SkylineGlobe/corporate/Default.aspx?) - 3D desktop and web-based applications, enabling an enterprise to build, edit, navigate, query, and analyze realistic 3D environments.
+- [World Wind](http://worldwind.arc.nasa.gov/java/) -  An SDK (software development kit) that software engineers can use to build their own applications
 
 
-## Web Map Server
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 3]
```diff
@@ -930,41 +927,37 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
     - [T-Drive trajectory data sample](http://research.microsoft.com/apps/pubs/default.aspx?id=152883)
-    - [pm2.5-China](http://www.pm25.in/)
     - [USGS Remote Sensing Image](http://earthexplorer.usgs.gov/)
-    - [metro extracts](https://mapzen.com/data/metro-extracts/) - City-sized portions of OpenStreetMap
-    - [Geofabrik](http://download.geofabrik.de/)
-    - [Geo Maps](https://github.com/simonepri/geo-maps) - High Quality GeoJSON maps programmatically generated.
+    - [WorldPop](http://www.worldpop.org.uk/)
 
 
-## News Site
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 4]
```diff
@@ -975,30 +968,30 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
 - [awesome-open-geoscience](https://github.com/softwareunderground/awesome-open-geoscience) - Curated from repositories that make our lives as geoscientists, hackers and data wranglers easier or just more awesome.
 
 
-## Amazing Map Site
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 5]
```diff
@@ -975,30 +968,30 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
-- [CHATTY MAPS](http://goodcitylife.org/chattymaps/index.html) - the sounds map of city.
+- [Odyssey.js](http://cartodb.github.io/odyssey.js/)
+- [OpenWebGIS is free online GIS](http://openwebgisystem.blogspot.com/)
 - [roads to rome](http://roadstorome.moovellab.com/) - roads to ROME
+- [snazzymaps](https://snazzymaps.com/) - A google map style gallery
+- [thematicmapping](http://blog.thematicmapping.org/)
 
 
 ## Other
-### Data Format
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 6]
```diff
@@ -975,30 +968,30 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
+### Data Formats
 - [Vector Formats](http://www.gdal.org/ogr_formats.html) -  GDAL vector formats: ESRI Shapefile, ESRI ArcSDE, ESRI FileGDB, MapInfo, GML, KML, PostGIS, Oracle Spatial, ...
 - [Raster Formats](http://www.gdal.org/formats_list.html) - GDAL raster formats: GeoTIFF, Erdas Imagine, ECW, MrSID, JPEG2000, DTED, NITF, ...
 
-### Design Material
```
Comment (sshuair) [inline README.md]: table of content link failed

File: README.md [hunk 7]
```diff
@@ -73,48 +73,46 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
-## Geographic Information System SoftWare
-- [ArcGIS](https://www.arcgis.com/) :star2: - An enterprise-level desktop GIS software published by Ersi
-- [QGIS](http://qgis.org/en/site/) :star2: - A cross-platform free and open-source desktop GIS software
-- [GeoDa](http://geodacenter.github.io/) - An Introduction to Spatial Data Analysis
-- [GRASS GIS](https://grass.osgeo.org/) - Used for geospatial data management and analysis, as a founding member of OSGEO
-- [uDig](http://udig.refractions.net/) - An open source desktop application framework built with Eclipse
-- [gvSIG](http://www.gvsig.com/en) - A powerful, user-friendly, interoperable geomatics professionals
-- [Marble](https://marble.kde.org/) - A virtual globe and world atlas
-- [SharpMap](https://github.com/SharpMap/SharpMap) - An mapping library in web and desktop applications built in .NET
-- [JUMP GIS](http://jump-pilot.sourceforge.net/) - An open source GIS written in Java
```
Comment (sshuair) [inline README.md]: This project is still an active project. We should keep it.

File: README.md (hunk 1 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 6 above)
Comment (csmatthews) [reply README.md]: Fixed

File: README.md (hunk 7 above)
Comment (csmatthews) [reply README.md]: Fixed. When I checked the URL the sourceforge page was taken down.

File: README.md [hunk 8]
```diff
@@ -60,61 +60,60 @@ Inspired by [Awesome Python](https://github.com/vinta/awesome-python).
...
+- [gvSIG](http://www.gvsig.com/en) - A powerful, user-friendly, interoperable GIS.
 - [JUMP GIS](http://jump-pilot.sourceforge.net/) - An open source GIS written in Java 
-- [Whitebox GAT](http://www.uoguelph.ca/~hydrogeo/Whitebox/) - Software package for geospatial analysis and data visualization
-- [TileMill](https://tilemill-project.github.io/tilemill/) - Creating beautiful interactive maps with CartoCSS. [Github Link](https://github.com/tilemill-project)
-- [Global Mapper](http://www.bluemarblegeo.com/products/global-mapper.php) - A geographic information system (GIS) software package currently developed by Blue Marble Geographics that runs on Microsoft Windows
 - [OpenOrienteering Mapper](https://github.com/openorienteering/mapper) - A software for creating maps for the orienteering sport.
-- [Mapinfo](https://www.pitneybowes.com/us/location-intelligence/geographic-information-systems/mapinfo-pro.html) - A desktop geographic information system software.
-- [SAGA](http://www.saga-gis.org/en/index.html) - System for automated geoscientific analyses.
```
Comment (sshuair) [inline README.md]: We should keep it.

Comment (csmatthews) [reply README.md]: Added back in, URL was down when I reviewed initially.

**Pull request merged**